lib/international_opt_in_people.rb

```diff
@@ -0,0 +1,13 @@
+module InternationalOptInPeople
```

Comment:
Is this only used in `Pd::InternationalOptIn`? Why does it need to be its own module? And why in the root `lib` directory? Do we expect to share with Pegasus? 

As [suggested](https://github.com/code-dot-org/code-dot-org/pull/23235#discussion_r198296006) earlier, if we put it instead in `dashboard/lib` we won't need [this require](https://github.com/code-dot-org/code-dot-org/pull/23235/files#diff-b940bb0be51d6330a2b3e0a9f53354feR16).